zombienet/config.toml

```diff
@@ -0,0 +1,20 @@
+[relaychain]
+default_command = "../polkadot/target/release/polkadot"
+default_args = [ "-lparachain=debug" ]
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
```

Comment:
I have `validator = true` set for both `relaychain.nodes` in my config.toml. Is that necessary? Should we add that flag here too?

Reply:
It is not mandatory. but i Added to config.toml file.

Reply:
Done.